interfaces/acados_template/acados_template/acados_ocp_solver.py

```diff
@@ -825,6 +831,10 @@ def __init__(self, acados_ocp, json_file='acados_ocp_nlp.json', simulink_opts=No
         assert getattr(self.shared_lib, f"{model.name}_acados_create")(self.capsule)==0
         self.solver_created = True
 
+        # create solver with another number of stages
```

Comment:
I think these lines are not needed, since the solver in Python is still created via
the line above:
```
         assert getattr(self.shared_lib, f"{model.name}_acados_create")(self.capsule)==0
```